src/widget.cpp

```diff
@@ -1374,12 +1374,12 @@ static std::string append_line( const std::string &line, bool first_row, int max
     int txt_w = 0;
     std::string txt;
     if( !line.empty() ) {
-        txt = skip_pad ? ( trim( line ) + ( newline_fix == 1 ? "\n" : "" ) ) : line;
+        txt = manual_align ? ( line + ( newline_fix == 1 ? "\n" : "" ) ) : line;
```

Comment:
```suggestion
        txt = line;
```
If you're doing away with `trim()`, you don't need to re-append the newline